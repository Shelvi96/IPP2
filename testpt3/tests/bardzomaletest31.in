NEW ID

50387>2715;5089;
916>171323658:
0>:
5>36:20
2916;>4640
;6754736>9932
71311642>2527
:>3992619205
98>27769
45237>:6485
:324>;
@4:5
@400;69;2655253894692
@3:92967;
@07;0:921;069240358
@62295;065759;4032:931:1:49
@;330
@627;675;6:0561083:3488706209
@4637521074:957768316:967
@:1:228703666;:5397042
@0
@7:894357849:48;72
@5173
@4:91485274340::;769925
@105715:29002888;74608428385
@:2132;:840:54281244:8:2464
@:49;6409:38670915;9625:01
@2
@6628237870563143614;1593:;73
