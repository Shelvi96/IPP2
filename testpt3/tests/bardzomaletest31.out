0
4796091
0
498120
69127952471688
0
30842277476216832
285993721838
860751360
0
13140
0
10329016323
2570189789684736
20011000000000
17848540206144
0
30842277476216832
